core/src/main/java/com/linecorp/armeria/server/Service.java

```diff
@@ -130,4 +132,12 @@ R decorate(Function<T, R> decorator) {
     default Service<I, O> decorate(DecoratingServiceFunction<I, O> function) {
         return new FunctionalDecoratingService<>(this, function);
     }
+
+    /**
+     * Returns whether the given {@code path} and {@code query }should be cached if the service's result is
```

Comment:
nit. Move whitespace before `should`
`{@code query }should`
->
`{@code query} should`

Reply:
Oops, thanks